Fixed the readme

Weltraumschaf opened the pull request.

Weltraumschaf committed:
#13 Add top-level headline

In Markdown it is highly recomended to add a top-level title.

Signed-off-by: Sven Strittmatter <sven.strittmatter@iteratec.com>

Weltraumschaf committed:
#13 Fix the term major

A mayor is the "boss" of a city ;-)

Signed-off-by: Sven Strittmatter <sven.strittmatter@iteratec.com>

Weltraumschaf committed:
#13 Fix path to scanner in readme

The direcotry for the scanner helm charts changed from `integrations`
to `scanner`.

Signed-off-by: Sven Strittmatter <sven.strittmatter@iteratec.com>

Weltraumschaf committed:
#13 Fix the hooks install guide

The hook add-attributes does not exist anymore, but instead there
is the update-field one missing.

Signed-off-by: Sven Strittmatter <sven.strittmatter@iteratec.com>

Weltraumschaf committed:
#13 Move comments out of example scripts

It is easier to read if the describing text is not inside the fenced
markdown blocks.

Signed-off-by: Sven Strittmatter <sven.strittmatter@iteratec.com>

Weltraumschaf committed:
#13 Fix spelling in install doc

In english almost everything but nouns are spelled lowercase.

Signed-off-by: Sven Strittmatter <sven.strittmatter@iteratec.com>

Weltraumschaf committed:
#13 Fix examples section in readme

All the examples were moved to the scanner direcotries.

Signed-off-by: Sven Strittmatter <sven.strittmatter@iteratec.com>

rfelber commented on README.md:
```diff
@@ -1,3 +1,5 @@
+# secureCodeBox – v2 ALPHA
```
I would like to remove the `ALPHA` because its more then that

rfelber commented on README.md:
````diff
@@ -56,61 +62,69 @@ There is a german article about [Security DevOps – Angreifern (immer) einen Sc
-# Deploy secureCodeBox Hooks 
-helm upgrade --install aah ./hooks/add-attributes/
+```bash
+helm upgrade --install aah ./hooks/update-field/
````
aah => ufh

rfelber commented on README.md:
````diff
@@ -56,61 +62,69 @@ There is a german article about [Security DevOps – Angreifern (immer) einen Sc
+Deploy the secureCodeBox operator first:
+
 ```bash
-# Deploy the secureCodeBox Operator first
````
Shouldn't we separate the bash block also for the "# Deploy SCB scanner Charts" headline too? Otherwise it's a little bit inconsistent 🤔

Weltraumschaf replied:
Done

Weltraumschaf committed:
#13 Move out bash comment as doc

For sake of consistency this one is pulled up from the fenced
bashe code, too.

Signed-off-by: Sven Strittmatter <sven.strittmatter@iteratec.com>

Weltraumschaf committed:
#13 Add an editor config

In coordination with the team added an editor config to stream line
and automate th indentation styles.

Signed-off-by: Sven Strittmatter <sven.strittmatter@iteratec.com>

Weltraumschaf committed:
#13 Only use one type of markdown lists

Signed-off-by: Sven Strittmatter <sven.strittmatter@iteratec.com>
